<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>frmRodadasAbrir</class>
 <widget class="QDialog" name="frmRodadasAbrir">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>987</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="windowIcon">
   <iconset resource="imgs/imagens.qrc">
    <normaloff>:/rodada.png</normaloff>:/rodada.png</iconset>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame_4">
     <property name="minimumSize">
      <size>
       <width>1</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="twgRodadas"/>
   </item>
   <item>
    <widget class="QFrame" name="frame">
     <property name="minimumSize">
      <size>
       <width>1</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_3">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>85</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QPushButton" name="btnSair">
      <property name="geometry">
       <rect>
        <x>900</x>
        <y>15</y>
        <width>55</width>
        <height>55</height>
       </rect>
      </property>
      <property name="toolTip">
       <string extracomment="Não abrir e Sair"/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="imgs/imagens.qrc">
        <normaloff>:/sair.png</normaloff>:/sair.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>50</width>
        <height>50</height>
       </size>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>840</x>
        <y>15</y>
        <width>55</width>
        <height>55</height>
       </rect>
      </property>
      <property name="toolTip">
       <string extracomment="Abrir a Rodada "/>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="icon">
       <iconset resource="imgs/imagens.qrc">
        <normaloff>:/rodada.png</normaloff>:/rodada.png</iconset>
      </property>
      <property name="iconSize">
       <size>
        <width>45</width>
        <height>45</height>
       </size>
      </property>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>16</x>
        <y>19</y>
        <width>391</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="lblRodadaSelecionada">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Selecione o Coordenador da Rodada</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="cmbIdCoordenador"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>19</y>
        <width>411</width>
        <height>48</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="lblRodadaSelecionada_2">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>Informe a Edição da Rodada</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="edtEdicaoRodada"/>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QFrame" name="frame_5">
     <property name="minimumSize">
      <size>
       <width>1</width>
       <height>0</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="imgs/imagens.qrc"/>
 </resources>
 <connections/>
</ui>
